make/release.mk

```
@@ -1,26 +1,39 @@
-IMAGE_BUILD_DIR := _build
-IMAGE_TAR_FILE := $(IMAGE_BUILD_DIR)/catalog-applications-image-bundle.tar.gz
-CATALOG_IMAGES_TXT := $(IMAGE_BUILD_DIR)/catalog_images.txt
+BUILD_DIR := _build
+IMAGE_TAR_FILE := $(BUILD_DIR)/catalog-applications-image-bundle.tar.gz
```

```suggestion
IMAGE_TAR_FILE := $(BUILD_DIR)/dkp-catalog-applications-image-bundle.tar.gz
```
do we want to prefix with dkp- to make it clear where these images are from?

---

yep done in 61b36c138e6a0920e6e3b265a1b4d966ce957e0b, this file will only be named this locally so the docs are still good